src/workloads/execution/VariadicAggregateExpressions.yml

```diff
@@ -0,0 +1,311 @@
+
+- ExpressionConcatArrays20:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionConcatArrays20
+      nopInPhasesUpTo: *MaxPhases
+      active: [8]
+      operationMetricsName: VariadicAggExpressionConcatArrays20
+      Pipeline: [{$project: {concatArrays: { $concatArrays: ["$arr1", "$arr2", "$arr3", "$arr4", "$arr5", "$arr6", "$arr7", "$arr8", "$arr9", "$arr10", "$arr11", "$arr12", "$arr13", "$arr14", "$arr15", "$arr16", "$arr17", "$arr18", "$arr19", "$arr20"]}}}]
+
+- ExpressionIfNull10:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionIfNull10
+      nopInPhasesUpTo: *MaxPhases
+      active: [9]
+      operationMetricsName: VariadicAggExpressionIfNull10
+      Pipeline: [{$project: {ifNull: { $ifNull: ["$arr1", "$arr2", "$arr3", "$arr4", "$arr5", "$arr6", "$arr7", "$arr8", "$arr9", "$arr10", 0]}}}]
+
+- ExpressionIfNull20:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionIfNull20
+      nopInPhasesUpTo: *MaxPhases
+      active: [10]
+      operationMetricsName: VariadicAggExpressionIfNull20
+      Pipeline: [{$project: {ifNull: { $ifNull: ["$arr1", "$arr2", "$arr3", "$arr4", "$arr5", "$arr6", "$arr7", "$arr8", "$arr9", "$arr10", "$arr11", "$arr12", "$arr13", "$arr14", "$arr15", "$arr16", "$arr17", "$arr18", "$arr19", "$arr20", 0]}}}]
+
+- ExpressionLet10:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionLet10
+      nopInPhasesUpTo: *MaxPhases
+      active: [11]
+      operationMetricsName: VariadicAggExpressionLet10
+      Pipeline: [{$project: {let: { $let: {vars: { a: "$int1", b: "$int2", c: "$int3", d: "$int4", e: "$int5", f: "$int6", g: "$int7", h: "$int8", i: "$int9", j: "$int10"}, in: {$add: ["$$a", "$$b", "$$c", "$$d", "$$e", "$$f", "$$g", "$$h", "$$i", "$$j"]}}}}}]
+
+- ExpressionLet20:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionLet20
+      nopInPhasesUpTo: *MaxPhases
+      active: [12]
+      operationMetricsName: VariadicAggExpressionLet20
+      Pipeline: [{$project: {let: { $let: {vars: { a: "$int1", b: "$int2", c: "$int3", d: "$int4", e: "$int5", f: "$int6", g: "$int7", h: "$int8", i: "$int9", j: "$int10", k: "$int11", l: "$int12", m: "$int13", n: "$int14", o: "$int15", p: "$int16", q: "$int17", r: "$int18", s: "$int19", t: "$int20"}, in: {$add: ["$$a", "$$b", "$$c", "$$d", "$$e", "$$f", "$$g", "$$h", "$$i", "$$j", "$$k", "$$l", "$$m", "$$n", "$$o", "$$p", "$$q", "$$r", "$$s", "$$t"]}}}}}]
+
+- ExpressionMultiply10:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionMultiply10
+      nopInPhasesUpTo: *MaxPhases
+      active: [13]
+      operationMetricsName: VariadicAggExpressionMultiply10
+      Pipeline: [{$project: {multiply: { $multiply: *IntFields10}}}]
+
+- ExpressionMultiply20:
```

Comment:
we should also have coverage for the set expressions ($setUnion/equals/intersection)

Reply:
Done.